istioctl/cmd/add-to-mesh.go

```diff
@@ -55,6 +53,10 @@ var (
 	crdFactory = createDynamicInterface
 )
 
+const (
+	ServiceAccountNameKey = "kubernetes.io/service-account.name"
```

Comment:
Also available in `k8s.io/api/core/v1/types.go`

Reply:
How did you find this? I even looked and couldn't. Kubernetes dependencies confuse me :slightly_smiling_face:

Reply:
grep the k8s.io/kubernetes repo and look for matches under staging/src/k8s.io/*

```bash
[10:39:03] ~/go/src/k8s.io/kubernetes
$ grep -rn kubernetes.io/service-account.name *
pkg/apis/core/types.go:4678:    // - Secret.Annotations["kubernetes.io/service-account.name"] - the name of the ServiceAccount the token identifies
pkg/apis/core/types.go:4684:    ServiceAccountNameKey = "kubernetes.io/service-account.name"
pkg/apis/core/validation/validation.go:4905:            // Only require Annotations[kubernetes.io/service-account.name]
staging/src/k8s.io/api/core/v1/types.go:5414:   // - Secret.Annotations["kubernetes.io/service-account.name"] - the name of the ServiceAccount the token identifies
staging/src/k8s.io/api/core/v1/types.go:5420:   ServiceAccountNameKey = "kubernetes.io/service-account.name"
```